app/js/components/orderEntry/PastOrder.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+export const PastOrder = props => (
+  <tr>
+    <td>
+      {format(props.auditInfo.dateCreated, 'DD/MM/YYYY | HH:MM:SS')}
+    </td>
+    <td>Inactive</td>
+    <td>
+      {props.drug.display}:
+      {props.dose ? ` ${props.dose}` : ""}
+      {props.doseUnits.display ? ` ${props.doseUnits.display}` : ""}
+      {props.frequency.display ? `, ${props.frequency.display}` : ""}
+      {props.route.display ? `, ${props.route.display}` : ""}
+      {props.duration ? `, for ${props.duration}` : ""}
+      {props.durationUnits.display ? ` ${props.durationUnits.display} total` : ""}
+      {props.dosingInstructions ? `, (${props.dosingInstructions})` : ""}
+      {(props.quantity && props.quantityUnits.display) ? `, (Dispense: ${props.quantity} ${props.quantityUnits.display})` : ""}
+    </td>
+    <td>&nbsp;&nbsp;<span className="icon-ban-circle" /></td>
+  </tr>
+);
+export default PastOrder;
```

Comment:
Could you kindly make use of destructuring for props?

Reply:
Fixed